eng/pipelines/templates/jobs/live.tests.yml

```diff
@@ -93,12 +93,22 @@ jobs:
           JdkVersion: $(JavaTestVersion)
 
       - ${{ if ne(parameters.DisableAzureResourceCreation, 'true') }}:
+        - pwsh: |
```

Comment:
I assume you plan to remove these debugging steps before merging?

Reply:
Yes... something in the template syntax is causing `${{ if paramters.UseFederatedAuth }}:` to evaluate to `true` when `UseFederatedAuth` is definitely false. This can be fixed by evaluating `${{ if eq('true', paramters.UseFederatedAuth) }}` ... This problematic behavior has not surfaced in other pipelines like this one: https://github.com/Azure/azure-sdk-for-net/pull/44742 

It may have something to do with how template syntax is evaluated.

Reply:
I suspect it is because the parameter doesn't have a type of boolean in this template.